src/main/resources/derived-value-config.yaml

```diff
@@ -432,12 +381,9 @@ derived-values:
         condition:
           logicalOperator: AND
           conditions:
-            - pageName: homeAddress
-              input: sameMailingAddress
+            - <<: *homeAddressSameAsMailing?
               value: "false"
-            - pageName: mailingAddressValidation
-              input: useEnrichedAddress
-              value: "true"
+            - *useEnrichedMailingAddress?
```

Comment:
@bencalegari Youre asking about here, right? Yeah, it doesn't technically need the explicit "true" since it's in the anchor, but it might be more readable/clear if it's written out - happy to make the change if you think it's better

Reply:
<img width="449" alt="Screen Shot 2021-04-28 at 1 44 36 PM" src="https://user-images.githubusercontent.com/4923102/116449160-e12dd400-a827-11eb-9420-f34328b55aad.png">
My question was why one condition needed the `<<` while the other didn't. Is it because `value` gets merged into it?

Reply:
Yup - `value: "false"` overwrites the `value: "true"`